grammar.js

```diff
@@ -138,17 +141,16 @@ module.exports = grammar({
 
     assert_statement: $ => seq(
       'assert',
-      $._expression,
-      repeat(seq(',', $._expression))
+      $.expression_list
     ),
 
-    expression_statement: $ => choice(
+    _expression_statement: $ => prec(PREC.expression_statement, choice(
       $._expression,
-      seq(commaSep1($._expression), optional(',')),
+      $.expression_list,
```

Comment:
This is going to affect us slightly in certain places, e.g. `print_statement`, where `expression_list` will now show up in the tree where it didn't before.